src/components/CharacterTab.js

```diff
@@ -184,7 +182,11 @@ export default function CharacterTab(props) {
     characterGrid.current.api.redrawRows()
     window.forceCharacterTabUpdate()
     Message.success('Successfully unequipped character')
-    global.relicsGrid.current.api.redrawRows()
+
+    // Char and Relic tab are never mounted together - this is unavailble
+    if (global.relicsGrid?.current?.api) {
```

Comment:
this could probably be deleted?  with refactor, that api will never be mounted when view CharacterTab